woocommerce/woocommerce
# Move the switch to classic shortcode block button to separate component

### Changes in the PR
- Move modal component file from `incompatible-extension-notice` component folder to `switch-to-classic-shortcode-button` folder.
- Rename modal component file to model-content component file.
- Move the switch to classic shortcode block button to separate component folder

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Related #42072 

We're updating the generic incompatibility notice content and adding the switch to classic block button to it. 
In this PR, we're only moving the switch to classic shortcode block button to separate component so that it can reused and added to the generic incompatibility notice.

PR: #42751 will take care of the updating the generic incompatibility notice content and adding the switch to classic block button to it.


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install any incompatible plugin like PayTm
2. Edit the Cart block page.
3. Focus on the Cart block. See the incompatibility notice in the sidebar. Click the "switch to classic cart" button.
4. Modal should open. Click switch. Block should be replaced.
5. In the snackbar bottom left, click "undo". Should revert back to blocks.
6. Repeat the steps for the Checkout Block page editor. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Move the switch to classic shortcode block button to separate component.

#### Comment 

</details>
